src/cmap/connection.ts

```diff
@@ -797,8 +762,25 @@ export class CryptoConnection extends Connection {
       }
     }
 
-    const response = await super.command<T>(ns, encrypted, options, undefined);
+    const encryptedResponse: MongoDBResponse = (await super.command<T>(
+      ns,
+      encrypted,
+      options,
+      (responseType ?? MongoDBResponse) as any
```

Comment:
Technically, I'm committing type crimes. I added a comment clarifying this, but it is not right that we are saying `command` of `T` and then passing in `MongoDBResponse` when responseType is nullish because now that may not match `T`. I can move to the type crime into the generic or here on the argument, but it will exist until we can only pass in a `responseType` without defaulting to a different subtype constraint. 

```
typeof MongoDBResponse' is assignable to the constraint of type 'T', but 'T' could be instantiated with a different subtype of constraint 'MongoDBResponseConstructor'
```

Reply:
Why can't we remove the generic on `super`?  then super.command() should resolve to a mongodbresponse instance.

If there is no response type provided, line 777 resolves to a plain document of type T.  So there wouldn't be TS issues?